plugins/CoreHome/Columns/Metrics/EvolutionMetric.php

```diff
@@ -101,6 +109,15 @@ public function compute(Row $row)
         $currentValue = $this->getMetric($row, $columnName);
         $pastValue = $pastRow ? $this->getMetric($pastRow, $columnName) : 0;
 
+        // Reduce past value proportionally to match the percent of the current period which is complete, if applicable
+        $ratio = $this->getRatio($this->currentData, $this->pastData, $row);
```

Comment:
The method `getRatio` is static and therefor should be called that way.
```suggestion
        $ratio = self::getRatio($this->currentData, $this->pastData, $row);
```